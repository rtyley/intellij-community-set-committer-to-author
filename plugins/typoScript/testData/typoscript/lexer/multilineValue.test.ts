myObject(
  some value
)

myObject.nextProperty (IGNORED_TEXT
  some other value ))
  and one more string with provocations{
    [GLOBAL]
  }
)

myObject3 (
 prop{
  p1 = 2
  p4 = 5
 }
)also IGNORED_TEXT
