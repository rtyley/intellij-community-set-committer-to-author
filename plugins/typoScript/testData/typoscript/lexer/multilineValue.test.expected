object path entity ('myObject')
( ('(')
WHITE_SPACE_WITH_NEW_LINE ('
  ')
multiline value ('some value')
WHITE_SPACE ('
')
) (')')
WHITE_SPACE_WITH_NEW_LINE ('

')
object path entity ('myObject')
. ('.')
object path entity ('nextProperty')
WHITE_SPACE (' ')
( ('(')
IGNORED_TEXT ('IGNORED_TEXT')
WHITE_SPACE ('
  ')
multiline value ('some other value ))')
WHITE_SPACE ('
  ')
multiline value ('and one more string with provocations{')
WHITE_SPACE ('
    ')
multiline value ('[GLOBAL]')
WHITE_SPACE ('
  ')
multiline value ('}')
WHITE_SPACE ('
')
) (')')
WHITE_SPACE_WITH_NEW_LINE ('

')
object path entity ('myObject3')
WHITE_SPACE (' ')
( ('(')
WHITE_SPACE_WITH_NEW_LINE ('
 ')
multiline value ('prop{')
WHITE_SPACE ('
  ')
multiline value ('p1 = 2')
WHITE_SPACE ('
  ')
multiline value ('p4 = 5')
WHITE_SPACE ('
 ')
multiline value ('}')
WHITE_SPACE ('
')
) (')')
IGNORED_TEXT ('also IGNORED_TEXT')
WHITE_SPACE_WITH_NEW_LINE ('
')